tests/functions/test_origin_response.py

```diff
@@ -0,0 +1,118 @@
+from cdn_lambda.functions.origin_response.origin_response import lambda_handler
+import pytest
+
+
+def test_origin_response_rpm():
+    event = {
+        "Records": [
+            {
+                "cf": {
+                    "request": {
+                        "headers": {
+                            "original-uri": [
+                                {
+                                    "key": "original-uri",
+                                    "value": "www.example.com/some/repo/some-rpm.rpm",
```

Comment:
I don't think this is accurate testdata. I had to look this up while reviewing another pending PR and I found that uri does not include the host portion, and always starts with "/".  Could this please be accurate too?

The main reason to care about it is to avoid misleading readers of this code, i.e. something looking at this test and trying to understand the code would be led to wrongly guess that original-uri includes hostname, when it actually won't (right?)

I know that the other test already has this mistake, but let's try to avoid that misunderstanding propagating any further.